Pod/Classes/LocationManager.swift

```diff
@@ -10,244 +10,278 @@
 import CoreLocation
 import PromiseKit
 
-public enum LocationManagerError: ErrorType {
-    case LocationServiceDisabled
-    case CannotFetchLocation
+public enum LocationManagerError: Error {
+    case locationServiceDisabled
+    case cannotFetchLocation
 }
 
-public enum LocationManagerAuthorizationError: ErrorType {
-    case KeyInPlistMissing
+public enum LocationManagerAuthorizationError: Error {
+    case keyInPlistMissing
 }
 
-public class LocationManager: NSObject, CLLocationManagerDelegate {
-    
-    public static let locationDidChangeAuthorizationStatusNotification = "locationDidChangeAuthorizationStatusNotification"
-    
-    public static let sharedManager = LocationManager()
+open class LocationManager: NSObject, CLLocationManagerDelegate {
+
+    typealias AuthorizationFulfillment = (CLAuthorizationStatus) -> Void
+
+    open static let locationDidChangeAuthorizationStatusNotification = "locationDidChangeAuthorizationStatusNotification"
+    open static let sharedManager = LocationManager()
 
     var currentLocation: CLLocation?
+    internal var lastKnownLocation: CLLocation?
     
-    private var locationRequests: [LocationRequest] = []
-    private var locationObservers: Set<LocationObserverItem> = []
-    
-    public static var locationObserversCount: Int {
+    fileprivate var locationRequests: [LocationRequest] = []
+    fileprivate var locationObservers: Set<LocationObserverItem> = []
+    fileprivate var askForLocationServicesFulfillments: [AuthorizationFulfillment] = []
+    fileprivate let locationManager = CLLocationManager()
+
+    open static var locationObserversCount: Int {
         return self.sharedManager.locationObserversCount
     }
-    public var locationObserversCount: Int {
+    open var locationObserversCount: Int {
         return self.locationObservers.count
     }
-    
-    private var askForLocationServicesFulfillments: [AuthorizationFulfillment] = []
-    typealias AuthorizationFulfillment = CLAuthorizationStatus -> Void
-
-    private let locationManager = CLLocationManager()
 
     public override init() {
+
         super.init()
+
```

Comment:
Question: are these spaces based on some swift design, or is it just for readability (subjective)?